.github/workflows/pr.yml

```diff
@@ -1,18 +1,26 @@
-name: Source code lint
+name: PR
 
 on:
   pull_request:
     branches: [main]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number || github.ref }}
+  cancel-in-progress: true
+
 jobs:
-  lint:
+  pr:
+    name: PR
```

Comment:
PR feels like too generic of a name and doesn't describe what the job is actually doing. going back to a name like `lint` would be much better in my opinion

Reply:
Changed back :)